SPEC.md

```diff
@@ -565,7 +565,10 @@ JSON Schema for the Container Runtime Manifest (container manifest)
     "apps": [
         {
             "app": "example.com/reduce-worker-1.0.0",
-            "imageID": "sha512-..."
+            "imageID": "sha512-...",
+            "mountPoints": [
+                 {"source": "work", "destination": "work"}
```

Comment:
I think we need to call out how duplicate destinations are handled...? if two images in a crm have the same mountPoint.name